apps/studio/sanity.config.ts

```diff
@@ -18,26 +18,30 @@ export default defineConfig({
   },
   document: {
     productionUrl: async (prev, { document, getClient }) => {
-      const configuredClient = getClient({ apiVersion: "2022-10-06" });
-      const host = window.location.href.includes("localhost")
-        ? "http://localhost:3000"
-        : "https://spor.vy.no";
+      try {
```

Comment:
Unrelated code change that was required in order to fix a weird edge case with slugs.